packages/@sfcc-bff/basketapi/src/api/schema/basketResolvers.ts

```diff
@@ -68,11 +68,11 @@ const getBasketProductCount = async (config: Config, context: AppContext) => {
         },
     });
 
-    apiBasket.productItems
-        ? apiBasket.productItems.forEach(product => {
-              basketProductCount += product.quantity || 0;
-          })
-        : 0;
+    if (apiBasket.productItems) {
```

Comment:
@gurpreetsainisalesforce this is what would have been caught during lint. The `: 0` is meaningless, because there is no assignment for the expression.